### PR TITLE
feat(eventing): event specific details for host events

### PR DESCRIPTION
**Pull request by datacore-vvarakantham**
Added target and uuid of the target to host event specific details.

### CODE REVIEW

File: apis/events/protobuf/v1/event.proto [hunk 1]
```diff
@@ -181,4 +181,6 @@ message NvmePathEventDetails {
 message HostInitiatorEventDetails {
   string host_nqn = 1;
   string subsystem_nqn = 2;
+  string target = 3;
```
Comment (tiagolobocastro) [inline apis/events/protobuf/v1/event.proto:185]: What is target here?

Comment (tiagolobocastro) [reply apis/events/protobuf/v1/event.proto:185]: I see now, replica or nexus;

Comment (tiagolobocastro) [reply apis/events/protobuf/v1/event.proto:185]: Please doc comment or use enum

**Review by tiagolobocastro — APPROVED** [review]

**Review by chriswldenyer — APPROVED** [review]
LGTM

**Pull request merged**